var/spack/repos/builtin/packages/clhep/package.py

```diff
@@ -26,13 +26,23 @@
 from spack import *
 
 
-class Clhep(Package):
+class Clhep(CMakePackage):
     """CLHEP is a C++ Class Library for High Energy Physics. """
     homepage = "http://proj-clhep.web.cern.ch/proj-clhep/"
     url      = "http://proj-clhep.web.cern.ch/proj-clhep/DISTRIBUTION/tarFiles/clhep-2.2.0.5.tgz"
-    list_url = "https://proj-clhep.web.cern.ch/proj-clhep/DISTRIBUTION/"
+    list_url = "https://proj-clhep.web.cern.ch/proj-clhep/"
+    list_depth = 3
```

Comment:
Yes, the default is 0, meaning Spack only searches that `list_url` for releases, it doesn't spider down through links.